lib/report-portal-client.js

```diff
@@ -372,11 +387,18 @@ class RPClient {
     launchObj.promiseFinish.then(
       () => {
         const url = ['launch', launchObj.realId, 'update'].join('/');
+        this.logDebug(`Update launch with tempId ${launchTempId}`, launchData);
         this.restClient.update(url, launchData, { headers: this.headers }).then(
           (response) => {
+            this.logDebug(
```

Comment:
I have a linter error here : 
![screenshot_05](https://user-images.githubusercontent.com/84175555/208609057-3a58d7e1-26ae-40ae-9272-580a65fbf31d.jpg)
Please, fix it.
The same error when I run npm run lint locally.

Reply:
Ok. I guess we need to understand why this error missed in the pipeline. I'll check.

Reply:
Fixed.